src/styles/forms.css

```
@@ -93,25 +144,64 @@
     color: var(--sl-color-green-high);
+    font-size: var(--sl-text-sm);
+    line-height: 1.5;
+  }
+
+  .form-alert::before {
+    content: "";
+    flex: none;
+    width: 1.15rem;
+    height: 1.15rem;
+    margin-top: 0.05rem;
+    background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)